config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,11 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider:              'AWS',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.